packages/rum-core/src/rawRumEvent.types.ts

```diff
@@ -79,6 +79,9 @@ export interface RawRumViewEvent {
     long_task: Count
     resource: Count
   }
+  session: {
+    has_replay: true | undefined
```

Comment:
Semantically it would be better to have a boolean?

Reply:
Yes but in practice, this field should not exist for backward and other SDK compatibility. If I want to select all events that don't have replay, using `@session.has_replay:false` would lead to incorrect results since not all SDKs implement this flag. Using `-@session.has_replay:true` is the way to go.

Using this weird typing forces us to ensure `has_replay` is never `false`.